anomalib/models/dfkde/torch_model.py

```diff
@@ -48,7 +49,7 @@ def __init__(
         self.threshold_steepness = threshold_steepness
         self.threshold_offset = threshold_offset
 
-        self.feature_extractor = FeatureExtractor(backbone=backbone, pre_trained=pre_trained, layers=layers).eval()
+        self.feature_extractor = FeatureExtractor(feature_extractor).eval()
```

Comment:
Would it be an idea to set the feature extractor modules to eval mode by default? I guess eval mode will be desired in most cases.

Reply:
My personal preferences for this is to not assume much and leave it to the user adding option -- I know, it would be yet a new param...

There are many papers on feature adaptation showing up recently and I think there is no consensus on rather to update the feature extractor or not, so IMO it's hard to assume a better default.